provider/azure/internal/imageutils/images.go

```diff
@@ -94,7 +94,7 @@ func SeriesImage(
 		publisher = centOSPublisher
 		offering = centOSOffering
 		switch series {
-		case "centos7", "centos8":
+		case "centos7", "centos8", "centos9": // TODO: this doesn't look right
```

Comment:
Can we test on Azure to be sure this all works still

Reply:
So it would work, the problem here is that centos support on Azure is pretty unofficial.

Reply:
We need to return to this to improve the image selection on Azure.